src/ansi-c/ansi_c_parser.h

```diff
@@ -35,14 +34,14 @@ class ansi_c_parsert:public parsert
       for_has_scope(false),
       ts_18661_3_Floatn_types(false)
   {
+    // set up global scope
+    scopes.clear();
+    scopes.push_back(scopet());
   }
 
-  virtual bool parse() override
-  {
-    return yyansi_cparse()!=0;
-  }
+  bool parse() override;
 
-  virtual void clear() override
+  void clear() override
```

Comment:
❓ Is this function still used anywhere?

Reply:
I will clean this up across the type hierarchy (removing just this override seems dangerous, as any use of `.clear()` would just invoke `parsert::clear()`, which would end up in incomplete cleanup) in a follow-up PR.